services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/ContactSyncRequestHandler.java

```diff
@@ -0,0 +1,71 @@
+package com.softgroup.profile.impl.handler;
+
+import com.softgroup.database.DataBase;
+import com.softgroup.profile.api.message.ContactSyncRequest;
+import com.softgroup.profile.api.message.ContactSyncResponse;
+import com.softgroup.profile.api.model.Contact;
+import com.softgroup.profile.api.router.ProfileRequestHandler;
+import com.softgroup.common.protocol.ActionHeader;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import com.softgroup.common.protocol.ResponseStatus;
+import com.softgroup.common.router.api.AbstractRequestHandler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+/**
+ * Created by alex on 24.02.17.
+ */
+
+@Component
+public class ContactSyncRequestHandler extends AbstractRequestHandler<ContactSyncRequest, ContactSyncResponse>
```

Comment:
@Alexey007gold It's nice that you trying to use database. But this is not the proper way to do it. I'll show how to use database on next lesson.